nmea.js

```diff
@@ -140,6 +140,43 @@ function radsToPositiveDeg(r) {
   return radsToDeg(toPositiveRadians(r))
 }
 
+function formatDatetime(datetime8601) {
+  if (datetime8601 && typeof datetime8601 === 'string' && datetime8601.length > 0) {
+    const datetime = new Date(datetime8601);
+
+    const hours = ('00' + datetime.getUTCHours()).slice(-2);
+    const minutes = ('00' + datetime.getUTCMinutes()).slice(-2);
+    const seconds = ('00' + datetime.getUTCSeconds()).slice(-2);
+
+    const day = ('00' + datetime.getUTCDate()).slice(-2);
+    const month = ('00' + (datetime.getUTCMonth() + 1)).slice(-2); // months from 1-12
+    const year = ('00' + datetime.getUTCFullYear()).slice(-2);
+    return {
+      hours: hours,
+      minutes: minutes,
+      seconds: seconds,
+      // NMEA0183 time format is hhmmss.ss
```

Comment:
Im confused with the .ss; In ZDA '.020' is added as constant. Can you advice? The current PR doesn't change the behavior with that regard.